src/js/vendor/Recollect.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import Script from 'react-load-script'
+
+class Recollect extends Component {
+
+  getRecollectWidgetName() {
+
+    let name;
+
+    switch(this.props.type) {
+      case 'Collection Schedule Lookup':
+        name = "calendar";
+        break;
+      case 'What do I do with...':
+      default:
+        name = "wizard";
+        break;
+    }
+
+    return name;
+  }
+
+  handleScriptLoad() {
+
+    const name = this.getRecollectWidgetName();
+
+    let script = document.createElement('script');
+        script.setAttribute('type', 'text/javascript');
+        script.text = `
+          window.loader = window.loader || new Recollect.Widget.Loader({
+              area: "Austin",
+              name: "${name}",
+              container: "#rCw"
+            });
+          window.loader.load();`;
+        document.getElementById('rCw').appendChild(script);
+  }
+
+  componentWillMount() {
+    // TODO: temp fix as current implementation of recollect app is not SPA friendly
+    if(window.Recollect) {
+      window.location.reload();
+    }
+  }
+
+  shouldComponentUpdate() {
+    return false;
```

Comment:
Do we need this still?

Reply:
This was recommended in the implementation docs for third party scripts in react so any parent updates wouldn't result in downstream repaints, which is different than a remount.
Really, none of these stateless components should be components -- they should just be functions that spat out JSX -- and this one would have a check and reload before it spat out the JSX. But that's a bigger refactor.